include/deal.II/fe/fe_values.h

```diff
@@ -313,6 +369,24 @@ namespace FEValuesViews
                                  std::vector<typename ProductType<gradient_type,typename InputVector::value_type>::type> &gradients) const;
 
     /**
+     * Same as above, but using a of vector local degree-of-freedom values.
+     *
+     * The DoF values typically would be obtained in the following way:
```

Comment:
Hmm... this contradicts what @bangerth requested earlier. I suppose I could link to, lets say, `FEValuesViews::[Scalar/Vector/Tensor/SymmetricTensor]::get_function_values_from_local_dof_values()` from each of the subsequent functions within a given View.